roles/vdm/tasks/postgres/postgres-multi-tenant-config.yaml

```diff
@@ -100,6 +100,25 @@
     - uninstall
     - update
 
+- name: postgres crunchy v5 - uncomment config for front-door or tls disabled 
+  replace:
+    path: "{{ role_path }}/templates/transformers/{{ 'platform-postgres' if role == 'default' else role }}-crunchy-tuning-transformer.yaml"
```

Comment:
Do we also need to have this setup for the `cds-postgres` server as well if it's been created?

Reply:
The same logic applies to 'cds-postgres'. The condition `{{ 'platform-postgres' if role == 'default' else role }}` handles that.

Reply:
Thanks for the clarification.